.github/workflows/tests.yaml

```diff
@@ -316,6 +316,24 @@ jobs:
           echo "The old one will disappear after 7 days."
       - name: Integration Tests
         run: bazel test tests/integration:asset tests/integration:credentials tests/integration:logging tests/integration:redis
+  goldens-lint:
```

Comment:
Trying to install 3.9 and run nox in the integration check resulted in weird errors about GLIBC version that I had trouble resolving. The integration tests already check the goldens match the output from the generator, so this just runs the noxfiles as present in `tests/integration/goldens/...`